caffe2/operators/conv_gradient_op.cc

```diff
@@ -16,35 +16,71 @@ class GetConvGradient : public GradientMakerBase {
 
     auto compute_dX = !argsHelper.GetSingleArgument<bool>("no_gradient_to_input", 0);
 
-    if (def_.input_size() == 3) {
+    auto engine = def_.engine();
+
+    // Use seperate gradient ops
+    if (engine == "CUDNN") {
```

Comment:
I actually already have that locally, thought it'd got added. I'll update in the morning